spec/requests/api/v1/activities_spec.rb

```diff
@@ -1,7 +1,69 @@
 require 'rails_helper'
 
-RSpec.describe "Api::V1::Activities", type: :request do
-  describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+RSpec.describe 'Api::V1::Activities', type: :request do
+  let(:role) { FactoryBot.create(:role, name: 'admin', description: 'admin user') }
+  let(:role2) { FactoryBot.create(:role, name: 'user', description: 'user user') }
+  let(:activity) { FactoryBot.create(:activity, name: 'name 0', content: 'content 0') }
+
+  let(:user) do
+    FactoryBot.create(:user, first_name: 'abc', last_name: 'abc', email: 'abc@abc.com', password: '123456', role: role)
+  end
+  
+  let(:user2) do
+    FactoryBot.create(:user, first_name: 'abc5', last_name: 'abc5', email: 'abc5@abc.com', password: '123456',
+                             role: role2)
+  end
+
+  let(:token) { AuthTokenService.call(user.id) }
+  let(:token2) { AuthTokenService.call(user2.id) }
+
+  it 'return status 201 to create activity' do
+    post '/api/v1/activities', params: { name: 'name 1', content: 'content 1' }, headers: { 'Authorization' => token.to_s }
```

Comment:
Se podría reducir código repetido usando let:
```suggestion
    post '/api/v1/activities', params: valid_params, headers: valid_headers
```